tests/test_ftp.py

```diff
@@ -239,3 +244,49 @@ def test_close(self, mock_ftp, mock_storage):
         file_.is_dirty = True
         file_.read()
         file_.close()
+
+
+class FTPTLSTest(TestCase):
+    def setUp(self):
+        self.storage = ftp.FTPStorage(location=URL_TLS)
+
+    def test_init_no_location(self):
```

Comment:
This test is already run in the base `FTPTest`, no need to duplicate.

Reply:
Removed `test_init_no_location(self)`